fahadmak/stackoverflow_clone
# feature(Answer): Create and List Answers functionality

**What does this PR do?**
This PR ensures that a user can create an answer and get a list of answers for a question.

**Description of Task to be completed?**
- create CreateAnswerFormForm
- create AnswerView
- create Answer model
- create url patterns
- create question_detail.html template

**What are the relevant Pivotal Tracker stories?**
[#167138015](https://www.pivotaltracker.com/n/projects/2364284/stories/167138015)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/fahadmak/stackoverflow_clone.git`
- `cd stackoverflow_clone`
- `git checkout ft-post-answers-167138015`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `run migrations using this command python manage.py migrate`
- `run python manage.py runserver 127.0.0.1:8000`
- `Go to this link in your browser 127.0.0.1:8000`
- `You will be redirected to the login page where you should log in`
- `Click on the 'Register Today' link to go to sign up page`
- `Try to sign up an account and the use it in the login page`
- `Click on a question to go to the question_detail page`
- `Create an answer in question detail page`

**Screenshots**
<img width="1021" alt="Screen Shot 2019-07-14 at 7 57 49 PM" src="https://user-images.githubusercontent.com/13882936/61296319-8f4ac400-a7e2-11e9-8486-6c447515e9af.png">
